[Error] SHELL is not properly set

Commands run with `distrobox enter -- <command>` do not reflect the user's `$SHELL`
A more concrete problem is, if you install a terminal emulator or any program that reads the shell, such as a text editor, or even `tmux`, then it'll have the wrong `$SHELL` value

**To Reproduce**
1. Set host OS $SHELL value

```shell
% echo $SHELL
/usr/bin/zsh
```

2. Run `distrobox assemble create`
3. Run `distrobox enter` with a specific command (on the host)

```
distrobox enter -- printenv | grep SHELL
SHELL=/bin/sh
```

**Expected behavior**

```
distrobox enter -- printenv | grep SHELL
SHELL=/usr/bin/zsh
```

NOTES:

- Running a naked `distrobox enter` DOES make `SHELL=/usr/bin/zsh` :)
- `chsh` does not fix this, in `init_hooks`
- This is especially annoying when trying to launch a terminal emulator packaged in the container, which reads it's default shell from `SHELL` (and `tmux`)

**Logs**
```
distrobox enter --verbose -n builderbox -- printenv | grep SHELL  
+ '[' -z builderbox ']'
+ '[' '!' -t 0 ']'
+ '[' '!' -t 1 ']'
+ headless=1
+ case "${container_manager}" in
+ command -v podman
+ container_manager=podman
+ command -v podman
+ '[' 1 -ne 0 ']'
+ container_manager='podman --log-level debug'
+ '[' 0 -ne 0 ']'
+ container_home=/home/pjh
+ container_path=/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin
+ unshare_groups=0
+ container_status=unknown
++ podman --log-level debug inspect --type container --format 'container_status={{.State.Status}};
	unshare_groups={{ index .Config.Labels "distrobox.unshare_groups" }};
	{{range .Config.Env}}{{if and (ge (len .) 5) (eq (slice . 0 5) "HOME=")}}container_home={{slice . 5 | printf "%q"}}{{end}}{{end}};
	{{range .Config.Env}}{{if and (ge (len .) 5) (eq (slice . 0 5) "PATH=")}}container_path={{slice . 5 | printf "%q"}}{{end}}{{end}}' builderbox
INFO[0000] podman filtering at log level debug          
DEBU[0000] Called inspect.PersistentPreRunE(podman --log-level debug inspect --type container --format container_status={{.State.Status}};
	unshare_groups={{ index .Config.Labels "distrobox.unshare_groups" }};
	{{range .Config.Env}}{{if and (ge (len .) 5) (eq (slice . 0 5) "HOME=")}}container_home={{slice . 5 | printf "%q"}}{{end}}{{end}};
	{{range .Config.Env}}{{if and (ge (len .) 5) (eq (slice . 0 5) "PATH=")}}container_path={{slice . 5 | printf "%q"}}{{end}}{{end}} builderbox) 
DEBU[0000] Using conmon: "/usr/bin/conmon"              
INFO[0000] Using sqlite as database backend             
DEBU[0000] Overriding graph root "/var/home/pjh/.local/share/containers/storage" with "/home/pjh/.local/share/containers/storage" from database 
DEBU[0000] Using graph driver overlay                   
DEBU[0000] Using graph root /home/pjh/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000/containers     
DEBU[0000] Using static dir /var/home/pjh/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /var/home/pjh/.local/share/containers/storage/volumes 
DEBU[0000] Using transient store: false                 
DEBU[0000] [graphdriver] trying provided driver "overlay" 
DEBU[0000] Cached value indicated that overlay is supported 
DEBU[0000] Cached value indicated that overlay is supported 
DEBU[0000] Cached value indicated that metacopy is not being used 
DEBU[0000] Cached value indicated that native-diff is usable 
DEBU[0000] backingFs=btrfs, projectQuotaSupported=false, useNativeDiff=true, usingMetacopy=false 
DEBU[0000] Initializing event backend journald          
DEBU[0000] Configured OCI runtime crun-vm initialization failed: no valid executable found for OCI runtime crun-vm: invalid argument 
DEBU[0000] Configured OCI runtime kata initialization failed: no valid executable found for OCI runtime kata: invalid argument 
DEBU[0000] Configured OCI runtime runsc initialization failed: no valid executable found for OCI runtime runsc: invalid argument 
DEBU[0000] Configured OCI runtime youki initialization failed: no valid executable found for OCI runtime youki: invalid argument 
DEBU[0000] Configured OCI runtime krun initialization failed: no valid executable found for OCI runtime krun: invalid argument 
DEBU[0000] Configured OCI runtime crun-wasm initialization failed: no valid executable found for OCI runtime crun-wasm: invalid argument 
DEBU[0000] Configured OCI runtime runc initialization failed: no valid executable found for OCI runtime runc: invalid argument 
DEBU[0000] Configured OCI runtime runj initialization failed: no valid executable found for OCI runtime runj: invalid argument 
DEBU[0000] Configured OCI runtime ocijail initialization failed: no valid executable found for OCI runtime ocijail: invalid argument 
DEBU[0000] Using OCI runtime "/usr/bin/crun"            
INFO[0000] Setting parallel job count to 49             
DEBU[0000] Looking up image "8cdc3aa94e6740620e463993254eb0396c7fa17883757b1a4b6cb99c2cb335d0" in local containers storage 
DEBU[0000] Trying "8cdc3aa94e6740620e463993254eb0396c7fa17883757b1a4b6cb99c2cb335d0" ... 
DEBU[0000] parsed reference into "[overlay@/home/pjh/.local/share/containers/storage+/run/user/1000/containers]@8cdc3aa94e6740620e463993254eb0396c7fa17883757b1a4b6cb99c2cb335d0" 
DEBU[0000] Found image "8cdc3aa94e6740620e463993254eb0396c7fa17883757b1a4b6cb99c2cb335d0" as "8cdc3aa94e6740620e463993254eb0396c7fa17883757b1a4b6cb99c2cb335d0" in local containers storage 
DEBU[0000] Found image "8cdc3aa94e6740620e463993254eb0396c7fa17883757b1a4b6cb99c2cb335d0" as "8cdc3aa94e6740620e463993254eb0396c7fa17883757b1a4b6cb99c2cb335d0" in local containers storage ([overlay@/home/pjh/.local/share/containers/storage+/run/user/1000/containers]@8cdc3aa94e6740620e463993254eb0396c7fa17883757b1a4b6cb99c2cb335d0) 
DEBU[0000] Successfully loaded 1 networks               
DEBU[0000] Called inspect.PersistentPostRunE(podman --log-level debug inspect --type container --format container_status={{.State.Status}};
	unshare_groups={{ index .Config.Labels "distrobox.unshare_groups" }};
	{{range .Config.Env}}{{if and (ge (len .) 5) (eq (slice . 0 5) "HOME=")}}container_home={{slice . 5 | printf "%q"}}{{end}}{{end}};
	{{range .Config.Env}}{{if and (ge (len .) 5) (eq (slice . 0 5) "PATH=")}}container_path={{slice . 5 | printf "%q"}}{{end}}{{end}} builderbox) 
DEBU[0000] Shutting down engines                        
INFO[0000] Received shutdown.Stop(), terminating!        PID=76790
+ eval 'container_status=running;
	unshare_groups=1;
	container_home="/home/pjh";
	container_path="/usr/local/bin:/usr/bin"'
++ container_status=running
++ unshare_groups=1
++ container_home=/home/pjh
++ container_path=/usr/local/bin:/usr/bin
+ '[' 0 -ne 0 ']'
+ '[' running = unknown ']'
+ '[' running '!=' running ']'
+ '[' 1 -eq 0 ']'
+ '[' 1 -eq 1 ']'
+ set -- -c '"$0" "$@"' -- printenv
+ set -- -s /bin/sh -c '"$0" "$@"' -- printenv
+ '[' 1 -eq 0 ']'
+ set -- pjh -s /bin/sh -c '"$0" "$@"' -- printenv
+ set -- su pjh -s /bin/sh -c '"$0" "$@"' -- printenv
++ generate_enter_command
++ result_command=exec
++ result_command='exec
		--interactive'
++ result_command='exec
		--interactive
		--detach-keys='
++ '[' 1 -eq 1 ']'
++ awk '{a[i++]=$0} END {for (j=i-1; j>=0;) print a[j--]}'
++ result_command='exec
		--interactive
		--detach-keys=
			--user=root'
++ '[' 1 -eq 0 ']'
++ '[' 0 -eq 0 ']'
++ workdir=/home/pjh/dotenv/containers
++ '[' -n '' ']'
++ result_command='exec
		--interactive
		--detach-keys=
			--user=root
		--workdir=/home/pjh/dotenv/containers'
++ result_command='exec
		--interactive
		--detach-keys=
			--user=root
		--workdir=/home/pjh/dotenv/containers
		--env=CONTAINER_ID=builderbox'
++ result_command='exec
		--interactive
		--detach-keys=
			--user=root
		--workdir=/home/pjh/dotenv/containers
		--env=CONTAINER_ID=builderbox
		--env=DISTROBOX_ENTER_PATH=/usr/bin/distrobox-enter'
++ set +o xtrace
++ '[' -n '' ']'
++ result_command='exec
		--interactive
		--detach-keys=
			--user=root
		--workdir=/home/pjh/dotenv/containers
		--env=CONTAINER_ID=builderbox
		--env=DISTROBOX_ENTER_PATH=/usr/bin/distrobox-enter
			--env=COLORTERM=truecolor
			--env=XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session1
			--env=HISTCONTROL=ignoredups
			--env=HISTSIZE=10000
			--env=I3SOCK=/run/user/1000/sway-ipc.1000.1851.sock
			--env=XDG_CONFIG_HOME=/home/pjh/.config
			--env=DESKTOP_SESSION=sway
			--env=XCURSOR_SIZE=24
			--env=GPG_TTY=/dev/pts/0
			--env=EDITOR=/usr/bin/nano
			--env=PWD=/home/pjh/dotenv/containers
			--env=XDG_SESSION_DESKTOP=sway
			--env=LOGNAME=pjh
			--env=XDG_SESSION_TYPE=wayland
			--env=LANG=en_US.UTF-8
			--env=LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=01;37;41:su=37;41:sg=30;43:ca=00:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.7z=01;31:*.ace=01;31:*.alz=01;31:*.apk=01;31:*.arc=01;31:*.arj=01;31:*.bz=01;31:*.bz2=01;31:*.cab=01;31:*.cpio=01;31:*.crate=01;31:*.deb=01;31:*.drpm=01;31:*.dwm=01;31:*.dz=01;31:*.ear=01;31:*.egg=01;31:*.esd=01;31:*.gz=01;31:*.jar=01;31:*.lha=01;31:*.lrz=01;31:*.lz=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.lzo=01;31:*.pyz=01;31:*.rar=01;31:*.rpm=01;31:*.rz=01;31:*.sar=01;31:*.swm=01;31:*.t7z=01;31:*.tar=01;31:*.taz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tgz=01;31:*.tlz=01;31:*.txz=01;31:*.tz=01;31:*.tzo=01;31:*.tzst=01;31:*.udeb=01;31:*.war=01;31:*.whl=01;31:*.wim=01;31:*.xz=01;31:*.z=01;31:*.zip=01;31:*.zoo=01;31:*.zst=01;31:*.avif=01;35:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.webp=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=01;36:*.au=01;36:*.flac=01;36:*.m4a=01;36:*.mid=01;36:*.midi=01;36:*.mka=01;36:*.mp3=01;36:*.mpc=01;36:*.ogg=01;36:*.ra=01;36:*.wav=01;36:*.oga=01;36:*.opus=01;36:*.spx=01;36:*.xspf=01;36:*~=00;90:*#=00;90:*.bak=00;90:*.crdownload=00;90:*.dpkg-dist=00;90:*.dpkg-new=00;90:*.dpkg-old=00;90:*.dpkg-tmp=00;90:*.old=00;90:*.orig=00;90:*.part=00;90:*.rej=00;90:*.rpmnew=00;90:*.rpmorig=00;90:*.rpmsave=00;90:*.swp=00;90:*.tmp=00;90:*.ucf-dist=00;90:*.ucf-new=00;90:*.ucf-old=00;90:
			--env=XDG_CURRENT_DESKTOP=sway
			--env=SWAYSOCK=/run/user/1000/sway-ipc.1000.1851.sock
			--env=WAYLAND_DISPLAY=wayland-1
			--env=XDG_SESSION_CLASS=user
			--env=TERM=foot
			--env=USER=pjh
			--env=DISPLAY=:0
			--env=SHLVL=3
			--env=XDG_SESSION_ID=2
			--env=XDG_RUNTIME_DIR=/run/user/1000
			--env=DEBUGINFOD_IMA_CERT_PATH=/etc/keys/ima:
			--env=LC_ALL=en_US.UTF-8
			--env=JOURNAL_STREAM=9:27664
			--env=DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
			--env=MAIL=/var/spool/mail/pjh
			--env=OLDPWD=/home/pjh
		--env=PATH=/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/local/bin:/usr/bin
		--env=XDG_DATA_DIRS=/home/pjh/.local/share/flatpak/exports/share:/var/lib/flatpak/exports/share:/usr/local/share:/usr/share
		--env=XDG_CACHE_HOME=/home/pjh/.cache
		--env=XDG_CONFIG_HOME=/home/pjh/.config
		--env=XDG_DATA_HOME=/home/pjh/.local/share
		--env=XDG_STATE_HOME=/home/pjh/.local/state
		--env=XDG_CONFIG_DIRS=/etc/xdg
		builderbox'
++ tr -d '\t'
++ printf '%s\n' 'exec
		--interactive
		--detach-keys=
			--user=root
		--workdir=/home/pjh/dotenv/containers
		--env=CONTAINER_ID=builderbox
		--env=DISTROBOX_ENTER_PATH=/usr/bin/distrobox-enter
			--env=COLORTERM=truecolor
			--env=XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session1
			--env=HISTCONTROL=ignoredups
			--env=HISTSIZE=10000
			--env=I3SOCK=/run/user/1000/sway-ipc.1000.1851.sock
			--env=XDG_CONFIG_HOME=/home/pjh/.config
			--env=DESKTOP_SESSION=sway
			--env=XCURSOR_SIZE=24
			--env=GPG_TTY=/dev/pts/0
			--env=EDITOR=/usr/bin/nano
			--env=PWD=/home/pjh/dotenv/containers
			--env=XDG_SESSION_DESKTOP=sway
			--env=LOGNAME=pjh
			--env=XDG_SESSION_TYPE=wayland
			--env=LANG=en_US.UTF-8
			--env=LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=01;37;41:su=37;41:sg=30;43:ca=00:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.7z=01;31:*.ace=01;31:*.alz=01;31:*.apk=01;31:*.arc=01;31:*.arj=01;31:*.bz=01;31:*.bz2=01;31:*.cab=01;31:*.cpio=01;31:*.crate=01;31:*.deb=01;31:*.drpm=01;31:*.dwm=01;31:*.dz=01;31:*.ear=01;31:*.egg=01;31:*.esd=01;31:*.gz=01;31:*.jar=01;31:*.lha=01;31:*.lrz=01;31:*.lz=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.lzo=01;31:*.pyz=01;31:*.rar=01;31:*.rpm=01;31:*.rz=01;31:*.sar=01;31:*.swm=01;31:*.t7z=01;31:*.tar=01;31:*.taz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tgz=01;31:*.tlz=01;31:*.txz=01;31:*.tz=01;31:*.tzo=01;31:*.tzst=01;31:*.udeb=01;31:*.war=01;31:*.whl=01;31:*.wim=01;31:*.xz=01;31:*.z=01;31:*.zip=01;31:*.zoo=01;31:*.zst=01;31:*.avif=01;35:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.webp=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=01;36:*.au=01;36:*.flac=01;36:*.m4a=01;36:*.mid=01;36:*.midi=01;36:*.mka=01;36:*.mp3=01;36:*.mpc=01;36:*.ogg=01;36:*.ra=01;36:*.wav=01;36:*.oga=01;36:*.opus=01;36:*.spx=01;36:*.xspf=01;36:*~=00;90:*#=00;90:*.bak=00;90:*.crdownload=00;90:*.dpkg-dist=00;90:*.dpkg-new=00;90:*.dpkg-old=00;90:*.dpkg-tmp=00;90:*.old=00;90:*.orig=00;90:*.part=00;90:*.rej=00;90:*.rpmnew=00;90:*.rpmorig=00;90:*.rpmsave=00;90:*.swp=00;90:*.tmp=00;90:*.ucf-dist=00;90:*.ucf-new=00;90:*.ucf-old=00;90:
			--env=XDG_CURRENT_DESKTOP=sway
			--env=SWAYSOCK=/run/user/1000/sway-ipc.1000.1851.sock
			--env=WAYLAND_DISPLAY=wayland-1
			--env=XDG_SESSION_CLASS=user
			--env=TERM=foot
			--env=USER=pjh
			--env=DISPLAY=:0
			--env=SHLVL=3
			--env=XDG_SESSION_ID=2
			--env=XDG_RUNTIME_DIR=/run/user/1000
			--env=DEBUGINFOD_IMA_CERT_PATH=/etc/keys/ima:
			--env=LC_ALL=en_US.UTF-8
			--env=JOURNAL_STREAM=9:27664
			--env=DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
			--env=MAIL=/var/spool/mail/pjh
			--env=OLDPWD=/home/pjh
		--env=PATH=/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/local/bin:/usr/bin
		--env=XDG_DATA_DIRS=/home/pjh/.local/share/flatpak/exports/share:/var/lib/flatpak/exports/share:/usr/local/share:/usr/share
		--env=XDG_CACHE_HOME=/home/pjh/.cache
		--env=XDG_CONFIG_HOME=/home/pjh/.config
		--env=XDG_DATA_HOME=/home/pjh/.local/share
		--env=XDG_STATE_HOME=/home/pjh/.local/state
		--env=XDG_CONFIG_DIRS=/etc/xdg
		builderbox'
+ cmd='builderbox
--env=XDG_CONFIG_DIRS=/etc/xdg
--env=XDG_STATE_HOME=/home/pjh/.local/state
--env=XDG_DATA_HOME=/home/pjh/.local/share
--env=XDG_CONFIG_HOME=/home/pjh/.config
--env=XDG_CACHE_HOME=/home/pjh/.cache
--env=XDG_DATA_DIRS=/home/pjh/.local/share/flatpak/exports/share:/var/lib/flatpak/exports/share:/usr/local/share:/usr/share
--env=PATH=/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/local/bin:/usr/bin
--env=OLDPWD=/home/pjh
--env=MAIL=/var/spool/mail/pjh
--env=DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
--env=JOURNAL_STREAM=9:27664
--env=LC_ALL=en_US.UTF-8
--env=DEBUGINFOD_IMA_CERT_PATH=/etc/keys/ima:
--env=XDG_RUNTIME_DIR=/run/user/1000
--env=XDG_SESSION_ID=2
--env=SHLVL=3
--env=DISPLAY=:0
--env=USER=pjh
--env=TERM=foot
--env=XDG_SESSION_CLASS=user
--env=WAYLAND_DISPLAY=wayland-1
--env=SWAYSOCK=/run/user/1000/sway-ipc.1000.1851.sock
--env=XDG_CURRENT_DESKTOP=sway
--env=LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=01;37;41:su=37;41:sg=30;43:ca=00:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.7z=01;31:*.ace=01;31:*.alz=01;31:*.apk=01;31:*.arc=01;31:*.arj=01;31:*.bz=01;31:*.bz2=01;31:*.cab=01;31:*.cpio=01;31:*.crate=01;31:*.deb=01;31:*.drpm=01;31:*.dwm=01;31:*.dz=01;31:*.ear=01;31:*.egg=01;31:*.esd=01;31:*.gz=01;31:*.jar=01;31:*.lha=01;31:*.lrz=01;31:*.lz=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.lzo=01;31:*.pyz=01;31:*.rar=01;31:*.rpm=01;31:*.rz=01;31:*.sar=01;31:*.swm=01;31:*.t7z=01;31:*.tar=01;31:*.taz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tgz=01;31:*.tlz=01;31:*.txz=01;31:*.tz=01;31:*.tzo=01;31:*.tzst=01;31:*.udeb=01;31:*.war=01;31:*.whl=01;31:*.wim=01;31:*.xz=01;31:*.z=01;31:*.zip=01;31:*.zoo=01;31:*.zst=01;31:*.avif=01;35:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.webp=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=01;36:*.au=01;36:*.flac=01;36:*.m4a=01;36:*.mid=01;36:*.midi=01;36:*.mka=01;36:*.mp3=01;36:*.mpc=01;36:*.ogg=01;36:*.ra=01;36:*.wav=01;36:*.oga=01;36:*.opus=01;36:*.spx=01;36:*.xspf=01;36:*~=00;90:*#=00;90:*.bak=00;90:*.crdownload=00;90:*.dpkg-dist=00;90:*.dpkg-new=00;90:*.dpkg-old=00;90:*.dpkg-tmp=00;90:*.old=00;90:*.orig=00;90:*.part=00;90:*.rej=00;90:*.rpmnew=00;90:*.rpmorig=00;90:*.rpmsave=00;90:*.swp=00;90:*.tmp=00;90:*.ucf-dist=00;90:*.ucf-new=00;90:*.ucf-old=00;90:
--env=LANG=en_US.UTF-8
--env=XDG_SESSION_TYPE=wayland
--env=LOGNAME=pjh
--env=XDG_SESSION_DESKTOP=sway
--env=PWD=/home/pjh/dotenv/containers
--env=EDITOR=/usr/bin/nano
--env=GPG_TTY=/dev/pts/0
--env=XCURSOR_SIZE=24
--env=DESKTOP_SESSION=sway
--env=XDG_CONFIG_HOME=/home/pjh/.config
--env=I3SOCK=/run/user/1000/sway-ipc.1000.1851.sock
--env=HISTSIZE=10000
--env=HISTCONTROL=ignoredups
--env=XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session1
--env=COLORTERM=truecolor
--env=DISTROBOX_ENTER_PATH=/usr/bin/distrobox-enter
--env=CONTAINER_ID=builderbox
--workdir=/home/pjh/dotenv/containers
--user=root
--detach-keys=
--interactive
exec'
+ IFS='
'
+ for arg in ${cmd}
+ set - builderbox su pjh -s /bin/sh -c '"$0" "$@"' -- printenv
INFO[0000] podman filtering at log level debug          
DEBU[0000] Called exec.PersistentPreRunE(podman --log-level debug exec --interactive --detach-keys= --user=root --workdir=/home/pjh/dotenv/containers --env=CONTAINER_ID=builderbox --env=DISTROBOX_ENTER_PATH=/usr/bin/distrobox-enter --env=COLORTERM=truecolor --env=XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session1 --env=HISTCONTROL=ignoredups --env=HISTSIZE=10000 --env=I3SOCK=/run/user/1000/sway-ipc.1000.1851.sock --env=XDG_CONFIG_HOME=/home/pjh/.config --env=DESKTOP_SESSION=sway --env=XCURSOR_SIZE=24 --env=GPG_TTY=/dev/pts/0 --env=EDITOR=/usr/bin/nano --env=PWD=/home/pjh/dotenv/containers --env=XDG_SESSION_DESKTOP=sway --env=LOGNAME=pjh --env=XDG_SESSION_TYPE=wayland --env=LANG=en_US.UTF-8 --env=LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=01;37;41:su=37;41:sg=30;43:ca=00:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.7z=01;31:*.ace=01;31:*.alz=01;31:*.apk=01;31:*.arc=01;31:*.arj=01;31:*.bz=01;31:*.bz2=01;31:*.cab=01;31:*.cpio=01;31:*.crate=01;31:*.deb=01;31:*.drpm=01;31:*.dwm=01;31:*.dz=01;31:*.ear=01;31:*.egg=01;31:*.esd=01;31:*.gz=01;31:*.jar=01;31:*.lha=01;31:*.lrz=01;31:*.lz=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.lzo=01;31:*.pyz=01;31:*.rar=01;31:*.rpm=01;31:*.rz=01;31:*.sar=01;31:*.swm=01;31:*.t7z=01;31:*.tar=01;31:*.taz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tgz=01;31:*.tlz=01;31:*.txz=01;31:*.tz=01;31:*.tzo=01;31:*.tzst=01;31:*.udeb=01;31:*.war=01;31:*.whl=01;31:*.wim=01;31:*.xz=01;31:*.z=01;31:*.zip=01;31:*.zoo=01;31:*.zst=01;31:*.avif=01;35:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.webp=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=01;36:*.au=01;36:*.flac=01;36:*.m4a=01;36:*.mid=01;36:*.midi=01;36:*.mka=01;36:*.mp3=01;36:*.mpc=01;36:*.ogg=01;36:*.ra=01;36:*.wav=01;36:*.oga=01;36:*.opus=01;36:*.spx=01;36:*.xspf=01;36:*~=00;90:*#=00;90:*.bak=00;90:*.crdownload=00;90:*.dpkg-dist=00;90:*.dpkg-new=00;90:*.dpkg-old=00;90:*.dpkg-tmp=00;90:*.old=00;90:*.orig=00;90:*.part=00;90:*.rej=00;90:*.rpmnew=00;90:*.rpmorig=00;90:*.rpmsave=00;90:*.swp=00;90:*.tmp=00;90:*.ucf-dist=00;90:*.ucf-new=00;90:*.ucf-old=00;90: --env=XDG_CURRENT_DESKTOP=sway --env=SWAYSOCK=/run/user/1000/sway-ipc.1000.1851.sock --env=WAYLAND_DISPLAY=wayland-1 --env=XDG_SESSION_CLASS=user --env=TERM=foot --env=USER=pjh --env=DISPLAY=:0 --env=SHLVL=3 --env=XDG_SESSION_ID=2 --env=XDG_RUNTIME_DIR=/run/user/1000 --env=DEBUGINFOD_IMA_CERT_PATH=/etc/keys/ima: --env=LC_ALL=en_US.UTF-8 --env=JOURNAL_STREAM=9:27664 --env=DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus --env=MAIL=/var/spool/mail/pjh --env=OLDPWD=/home/pjh --env=PATH=/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/local/bin:/usr/bin --env=XDG_DATA_DIRS=/home/pjh/.local/share/flatpak/exports/share:/var/lib/flatpak/exports/share:/usr/local/share:/usr/share --env=XDG_CACHE_HOME=/home/pjh/.cache --env=XDG_CONFIG_HOME=/home/pjh/.config --env=XDG_DATA_HOME=/home/pjh/.local/share --env=XDG_STATE_HOME=/home/pjh/.local/state --env=XDG_CONFIG_DIRS=/etc/xdg builderbox su pjh -s /bin/sh -c "$0" "$@" -- printenv) 
DEBU[0000] Using conmon: "/usr/bin/conmon"              
INFO[0000] Using sqlite as database backend             
DEBU[0000] Overriding graph root "/var/home/pjh/.local/share/containers/storage" with "/home/pjh/.local/share/containers/storage" from database 
DEBU[0000] Using graph driver overlay                   
DEBU[0000] Using graph root /home/pjh/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000/containers     
DEBU[0000] Using static dir /var/home/pjh/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /var/home/pjh/.local/share/containers/storage/volumes 
DEBU[0000] Using transient store: false                 
DEBU[0000] [graphdriver] trying provided driver "overlay" 
DEBU[0000] Cached value indicated that overlay is supported 
DEBU[0000] Cached value indicated that overlay is supported 
DEBU[0000] Cached value indicated that metacopy is not being used 
DEBU[0000] Cached value indicated that native-diff is usable 
DEBU[0000] backingFs=btrfs, projectQuotaSupported=false, useNativeDiff=true, usingMetacopy=false 
DEBU[0000] Initializing event backend journald          
DEBU[0000] Configured OCI runtime crun-vm initialization failed: no valid executable found for OCI runtime crun-vm: invalid argument 
DEBU[0000] Configured OCI runtime crun-wasm initialization failed: no valid executable found for OCI runtime crun-wasm: invalid argument 
DEBU[0000] Configured OCI runtime runsc initialization failed: no valid executable found for OCI runtime runsc: invalid argument 
DEBU[0000] Configured OCI runtime youki initialization failed: no valid executable found for OCI runtime youki: invalid argument 
DEBU[0000] Configured OCI runtime krun initialization failed: no valid executable found for OCI runtime krun: invalid argument 
DEBU[0000] Configured OCI runtime ocijail initialization failed: no valid executable found for OCI runtime ocijail: invalid argument 
DEBU[0000] Configured OCI runtime runc initialization failed: no valid executable found for OCI runtime runc: invalid argument 
DEBU[0000] Configured OCI runtime runj initialization failed: no valid executable found for OCI runtime runj: invalid argument 
DEBU[0000] Configured OCI runtime kata initialization failed: no valid executable found for OCI runtime kata: invalid argument 
DEBU[0000] Using OCI runtime "/usr/bin/crun"            
INFO[0000] Setting parallel job count to 49             
INFO[0000] Created exec session 5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278 in container 71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290 
INFO[0000] Going to start container 71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290 exec session 5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278 and attach to it 
DEBU[0000] /usr/bin/conmon messages will be logged to syslog 
DEBU[0000] running conmon: /usr/bin/conmon               args="[--api-version 1 -c 71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290 -u 5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278 -r /usr/bin/crun -b /home/pjh/.local/share/containers/storage/overlay-containers/71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290/userdata/5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278 -p /home/pjh/.local/share/containers/storage/overlay-containers/71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290/userdata/5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278/exec_pid -n builderbox --exit-dir /home/pjh/.local/share/containers/storage/overlay-containers/71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290/userdata/5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278/exit --persist-dir /home/pjh/.local/share/containers/storage/overlay-containers/71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290/userdata/5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278/persist/71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290 --full-attach -s -l none --log-level debug --syslog -i -e --exec-attach --exec-process-spec /home/pjh/.local/share/containers/storage/overlay-containers/71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290/userdata/5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278/exec-process-1992277651 --exit-command /usr/bin/podman --exit-command-arg --root --exit-command-arg /home/pjh/.local/share/containers/storage --exit-command-arg --runroot --exit-command-arg /run/user/1000/containers --exit-command-arg --log-level --exit-command-arg debug --exit-command-arg --cgroup-manager --exit-command-arg systemd --exit-command-arg --tmpdir --exit-command-arg /run/user/1000/libpod/tmp --exit-command-arg --network-config-dir --exit-command-arg  --exit-command-arg --network-backend --exit-command-arg netavark --exit-command-arg --volumepath --exit-command-arg /var/home/pjh/.local/share/containers/storage/volumes --exit-command-arg --db-backend --exit-command-arg sqlite --exit-command-arg --transient-store=false --exit-command-arg --runtime --exit-command-arg crun --exit-command-arg --storage-driver --exit-command-arg overlay --exit-command-arg --events-backend --exit-command-arg journald --exit-command-arg --syslog --exit-command-arg container --exit-command-arg cleanup --exit-command-arg --stopped-only --exit-command-arg --exec --exit-command-arg 5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278 --exit-command-arg 71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290]"
INFO[0000] Running conmon under slice user.slice and unitName libpod-conmon-71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290.scope 
DEBU[0000] Attaching to container 71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290 exec session 5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278 
DEBU[0000] Received: 0                                  
DEBU[0000] Received: 76873                              
[conmon:d]: exec with attach is waiting for start message from parent
[conmon:d]: exec with attach got start message from parent
DEBU[0000] Successfully started exec session 5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278 in container 71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290 
SHELL=/bin/sh
DEBU[0000] Container 71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290 exec session 5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278 completed with exit code 0 
DEBU[0000] Checking status of container 71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290 exec session 5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278 
INFO[0000] Removing container 71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290 exec session 5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278 
DEBU[0000] Successfully removed container 71790abd12c4d34517ab346757a1c8c7206f17afa58c36546aae1c6b9251e290 exec session 5b7f5bb528bcbbd77946436fe3ac8b7aafb28e41ed5d66af3e1ab3da628c6278 
DEBU[0000] Called exec.PersistentPostRunE(podman --log-level debug exec --interactive --detach-keys= --user=root --workdir=/home/pjh/dotenv/containers --env=CONTAINER_ID=builderbox --env=DISTROBOX_ENTER_PATH=/usr/bin/distrobox-enter --env=COLORTERM=truecolor --env=XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session1 --env=HISTCONTROL=ignoredups --env=HISTSIZE=10000 --env=I3SOCK=/run/user/1000/sway-ipc.1000.1851.sock --env=XDG_CONFIG_HOME=/home/pjh/.config --env=DESKTOP_SESSION=sway --env=XCURSOR_SIZE=24 --env=GPG_TTY=/dev/pts/0 --env=EDITOR=/usr/bin/nano --env=PWD=/home/pjh/dotenv/containers --env=XDG_SESSION_DESKTOP=sway --env=LOGNAME=pjh --env=XDG_SESSION_TYPE=wayland --env=LANG=en_US.UTF-8 --env=LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=01;37;41:su=37;41:sg=30;43:ca=00:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.7z=01;31:*.ace=01;31:*.alz=01;31:*.apk=01;31:*.arc=01;31:*.arj=01;31:*.bz=01;31:*.bz2=01;31:*.cab=01;31:*.cpio=01;31:*.crate=01;31:*.deb=01;31:*.drpm=01;31:*.dwm=01;31:*.dz=01;31:*.ear=01;31:*.egg=01;31:*.esd=01;31:*.gz=01;31:*.jar=01;31:*.lha=01;31:*.lrz=01;31:*.lz=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.lzo=01;31:*.pyz=01;31:*.rar=01;31:*.rpm=01;31:*.rz=01;31:*.sar=01;31:*.swm=01;31:*.t7z=01;31:*.tar=01;31:*.taz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tgz=01;31:*.tlz=01;31:*.txz=01;31:*.tz=01;31:*.tzo=01;31:*.tzst=01;31:*.udeb=01;31:*.war=01;31:*.whl=01;31:*.wim=01;31:*.xz=01;31:*.z=01;31:*.zip=01;31:*.zoo=01;31:*.zst=01;31:*.avif=01;35:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.webp=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=01;36:*.au=01;36:*.flac=01;36:*.m4a=01;36:*.mid=01;36:*.midi=01;36:*.mka=01;36:*.mp3=01;36:*.mpc=01;36:*.ogg=01;36:*.ra=01;36:*.wav=01;36:*.oga=01;36:*.opus=01;36:*.spx=01;36:*.xspf=01;36:*~=00;90:*#=00;90:*.bak=00;90:*.crdownload=00;90:*.dpkg-dist=00;90:*.dpkg-new=00;90:*.dpkg-old=00;90:*.dpkg-tmp=00;90:*.old=00;90:*.orig=00;90:*.part=00;90:*.rej=00;90:*.rpmnew=00;90:*.rpmorig=00;90:*.rpmsave=00;90:*.swp=00;90:*.tmp=00;90:*.ucf-dist=00;90:*.ucf-new=00;90:*.ucf-old=00;90: --env=XDG_CURRENT_DESKTOP=sway --env=SWAYSOCK=/run/user/1000/sway-ipc.1000.1851.sock --env=WAYLAND_DISPLAY=wayland-1 --env=XDG_SESSION_CLASS=user --env=TERM=foot --env=USER=pjh --env=DISPLAY=:0 --env=SHLVL=3 --env=XDG_SESSION_ID=2 --env=XDG_RUNTIME_DIR=/run/user/1000 --env=DEBUGINFOD_IMA_CERT_PATH=/etc/keys/ima: --env=LC_ALL=en_US.UTF-8 --env=JOURNAL_STREAM=9:27664 --env=DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus --env=MAIL=/var/spool/mail/pjh --env=OLDPWD=/home/pjh --env=PATH=/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/local/bin:/usr/bin --env=XDG_DATA_DIRS=/home/pjh/.local/share/flatpak/exports/share:/var/lib/flatpak/exports/share:/usr/local/share:/usr/share --env=XDG_CACHE_HOME=/home/pjh/.cache --env=XDG_CONFIG_HOME=/home/pjh/.config --env=XDG_DATA_HOME=/home/pjh/.local/share --env=XDG_STATE_HOME=/home/pjh/.local/state --env=XDG_CONFIG_DIRS=/etc/xdg builderbox su pjh -s /bin/sh -c "$0" "$@" -- printenv) 
DEBU[0000] Shutting down engines                        
INFO[0000] Received shutdown.Stop(), terminating!        PID=76780
```

**Desktop (please complete the following information):**

- Are you using podman, docker or lilipod?
  `podman`
- Which version or podman, docker or lilipod?
  `5.4.2`
- Which version of distrobox?
  `1.8.1.2`
- Which host distribution?
  `Fedora Sway Atomic`
- How did you install distrobox?
  `rpm-ostree` on host Fedora Sway Atomic

**Additional context**

Really, I want to get `ghostty` (and other apps inside the container) to launch with the correct `SHELL`
when running `distrobox enter -- ghostty` as found in distrobox exported: `.local/share/applications/builderbox-com.mitchellh.ghostty.desktop`

```
Exec=/usr/bin/distrobox-enter  -n builderbox  --   ghostty 
```

mb, I meant:
```
if [ "${unshare_groups:-0}" -eq 1 ]; then
	# shellcheck disable=SC2089,SC2016
	set -- "-c" '"$0" "$@"' -- "$@"
	set -- "-s" "/bin/sh" "$@"
	if [ "${headless}" -eq 0 ]; then
		set -- "--pty" "$@"
	fi
	set -- "-m" "$@"
	set -- "${container_command_user}" "$@"
	set -- "su" "$@"
fi
```
with distrobox version `1.8.2.1`.

--- a/distrobox-enter
+++ b/distrobox-enter
@@ -703,6 +703,7 @@ if [ "${unshare_groups:-0}" -eq 1 ]; then
 	if [ "${headless}" -eq 0 ]; then
 		set -- "--pty" "$@"
 	fi
+	set -- "-m" "$@"
 	set -- "${container_command_user}" "$@"
 	set -- "su" "$@"
 fi

